### ISSUE TITLE
Making categories

### ISSUE THREAD

**Issue by asmabrazi** [fosjump/fosjump#10]
On the long term there will be probably many links => a long list to check to find what attract us may be boring  , i suggest to make categories sorted by theme (medical , mechanical..) in the future in order to facilitate the research.

**Comment by SamyMe** [comment]
Good point ! It will inevitably get bigger indeed, and you're probably right for the categories part. I will think about that. Don't hesitate to doodle something if you find the time to :)

**Comment by asmabrazi** [comment]
Glad you're interested :) i'll try to still useful !
